keiko-core/build.gradle

```diff
@@ -5,5 +5,7 @@ dependencies {
   api "com.fasterxml.jackson.core:jackson-databind"
   api "org.springframework:spring-context"
 
+  implementation "com.netflix.spectator:spectator-api"
```

Comment:
keiko-core now depends on spectator, is that a problem?

Reply:
Is Spectator open source?

Reply:
yes, it's also not specific to Atlas. I'm just mindful of new dependencies, especially in `*-core`